ftp/1.7/modules/ROOT/pages/ftp-documentation.adoc

```diff
@@ -484,7 +484,7 @@ You can also use a matcher for additional filtering of files.
 | Field | Type | Description | Default Value | Required
 | Frequency a| Number | How often in milliseconds to reconnect. | |
 | Count a| Number | How many reconnection attempts to make. | |
-| blocking |Boolean |If false, the reconnection strategy runs in a separate, non-blocking thread. |true |
+| blocking |Boolean |If false, the reconnection strategy runs in a separate, non-blocking thread. Note: configure this parameter only in XML. |true |
```

Comment:
I know this is passive, but in this case, I think it's better to say it this way because the focus is on the field (if that makes sense):

This field is configurable only in XML. 

I always use this syntax for notes - NOTES: - but I don't remember how this renders in a table.